src/Wavefront.OpenTracing.SDK.CSharp/WavefrontSpan.cs

```diff
@@ -185,6 +186,16 @@ private WavefrontSpan SetTagObject(string key, object value)
                 {
                     forceSampling = (int)value > 0;
                     spanContext = spanContext.WithSamplingDecision(forceSampling.Value);
+                }
+
+                // allow span to be reported if debug is set to true.
+                if (forceSampling == null || !forceSampling.Value)
+                {
+                    if (DebugTagKey.Equals(key) && value.ToString().Equals("true"))
```

Comment:
Boolean.ToString() results in "True" or "False", so let's make this check case-insensitive (or convert to lowercase).

Reply:
hmm thats strange because the newly added unit test had the tag set as ("debug", true) and it passed. I modified the check anyways to make sure its in lower case.